src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/AbstractAdapter.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Institute for the Architecture of Application System - University of Stuttgart
+ * Author: Ghareeb Falazi
+ *
+ * This program and the accompanying materials are made available under the
+ * terms the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *******************************************************************************/
+package blockchains.iaas.uni.stuttgart.de.plugin.bitcoin;
+
+import blockchains.iaas.uni.stuttgart.de.api.interfaces.BlockchainAdapter;
+import blockchains.iaas.uni.stuttgart.de.api.interfaces.FinalityConfidenceCalculator;
+
+public abstract class AbstractAdapter implements BlockchainAdapter {
```

Comment:
I think it doesn't sense to have this class when we have only one adapter per plugin. Therefore, please remove it and move its contents to the `BitcoinAdapter` class